base/Dockerfile

```diff
@@ -100,6 +100,20 @@ RUN apk add --no-cache \
     && apk del .build-dependencies \
     && rm -rf /usr/src/arp-scan
 
+# PicoTTS - it has no specific version - commit should be taken from build.json
+ARG PICOTTS_HASH
+RUN apk add --no-cache --virtual .build-dependencies automake autoconf libtool popt-dev build-base \ 

```

Comment:
`popt-dev` your compiled software will not link against this library?

Reply:
Fixed with latest commit after rebase

Reply:
Thought that popt uses static-linking in this case, but I looked it up and you're right.